gef.py

```diff
@@ -2951,6 +2956,36 @@ def exit_handler(event):
         __gef_remote__ = None
     return
 
+def load_libc_args():
+    # load libc function arguments' definitions
+    if not get_gef_setting("context.libc_args"):
+        return
+
+    path = get_gef_setting("context.libc_args_path")
+    if path is None:
```

Comment:
I think we might want `if not path`. Is it possible for the setting to be set to an empty string?

Reply:
it's possible to have it as "", i'm handling this using os.path.isdir and printing a proper warn in case